scripts/build-docs.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# build_docs.sh updates the docs generated from this repo
+set -o nounset
+set -o errexit
+set -o pipefail
+readonly ROOT=$(dirname "${0}")/..
+
+if [[ "${TRAVIS_PUBLISH_PACKAGES:-}" == "true" ]] && [[ ! -z "${TRAVIS_TAG:-}" ]]; then
+    NPM_VERSION=$("${ROOT}/scripts/get-version")
+    VERSION=${NPM_VERSION#"v"}
+
+    echo "Building SDK docs for version ${VERSION}:"
+
+    # Clone the repo and fetch its dependencies.
+    # git clone "https://github.com/pulumi/docs.git" "$(go env GOPATH)/src/github.com/pulumi/docs"
```

Comment:
Did you mean to have this commented out?  I'm guessing not and it was an artifact of you testing things locally.

Reply:
Fixed.